v0/destinations/posthog/data/PHGroupConfig.json

```diff
@@ -36,5 +36,15 @@
     "destKey": "messageId",
     "sourceKeys": "messageId",
     "required": false
+  },
+  {
+    "destKey": "properties.$group_set",
+    "sourceKeys": "traits.groupProperties",
+    "required": false
+  },
+  {
+    "destKey": "properties.$group_type",
+    "sourceKeys": "traits.groupType",
+    "required": false
```

Comment:
If we are mapping traits to group_set, need to remove this groupType from traits. Check this from transformer.js.